pom.xml

```diff
@@ -67,6 +67,12 @@
             <artifactId>config</artifactId>
             <version>1.3.2</version>
         </dependency>
+        <!-- emoji java because i'm lazy -->
+        <dependency>
+            <groupId>com.vdurmont</groupId>
+            <artifactId>emoji-java</artifactId>
+            <version>5.1.1</version>
+        </dependency>
```

Comment:
I don't really think we need another dependency just for validation; I think overall the best bet for validation is just to attempt to add the reaction to a message, as the response code will clearly tell us if it's a valid emoji for discord and if we have permissions to use said emoji.

Reply:
Yes indeed, i didn't wanna put the DB under extra load for invalid/briefly unchecked user inputs tho, nor just send garbage straight to discord.